app/src/test/java/com/jalasoft/routesapp/ui/auth/login/viewModel/LoginViewModelTest.kt

```diff
@@ -48,6 +49,68 @@ class LoginViewModelTest : TestCase() {
         viewModel = LoginViewModel(fakeManager)
     }
 
+    @Test
+    fun `Given an empty or invalid format of email`() {
+        val emailTwo = "example"
+        val passwordTwo = "test.test"
+        runBlocking {
+            viewModel.validateFields(email, password)
+            val result = fakeManager.signInWithEmailAndPassword(email, password)
+            val resultTwo = fakeManager.signInWithEmailAndPassword(emailTwo, passwordTwo)
+            assertNotSame(result,resultTwo)
+            val observer = Observer<Boolean> {}
+            try {
+                viewModel.loginIsSuccessful.observeForever(observer)
+                val value = viewModel.loginIsSuccessful.getOrAwaitValue()
+                assertTrue(value)
```

Comment:
If the test is given a invalid email it should fail in loggin

Reply:
Corrected